code_samples/background_tasks/src/MessageHandler/SomeHandler.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace App\MessageHandler;
+
+use App\Message\SomeMessage;
+
+final class SomeHandler
```

Comment:
<img width="859" height="514" alt="Screenshot 2025-10-01 at 18 07 01" src="https://github.com/user-attachments/assets/b9c1732f-aae3-4a0d-9939-d7dc9a262050" />

Asking just to be sure: Symfony doc mentions that if you choose to avoid using the Attribute, then the `MessageHandlerInterface` interface needs to be implemented.

Can we skip it in this case?